Version 2.18 won't start

Version 2.18 of the Docker image seems broken.  It is outputting the following on the console, then dying: 

```
ls: BOOT-INF/lib/*.jar: No such file or directory
Error: Unable to initialize main class zipkin.server.ZipkinServer
Caused by: java.lang.NoClassDefFoundError: org/springframework/boot/Banner
```

--- a/zipkin-slim/zipkin/run.sh
+++ b/zipkin-slim/zipkin/run.sh
@@ -4,4 +4,4 @@ if [ -f ".${STORAGE_TYPE}_profile" ]; then
   source ${PWD}/.${STORAGE_TYPE}_profile
 fi
 
-exec java ${JAVA_OPTS} -cp .:$(ls BOOT-INF/lib/*.jar|tr '\n' ':')BOOT-INF/classes zipkin.server.ZipkinServer
+exec java ${JAVA_OPTS} -cp .:$(ls ${PWD}/BOOT-INF/lib/*.jar|tr '\n' ':')${PWD}/BOOT-INF/classes zipkin.server.ZipkinServer

--- a/zipkin/zipkin/run.sh
+++ b/zipkin/zipkin/run.sh
@@ -7,7 +7,7 @@ fi
 # Use main class directly if there are no modules, as it measured 14% faster from JVM running to available
 # verses PropertiesLauncher when using Zipkin was based on Spring Boot 2.1
 if [[ -z "$MODULE_OPTS" ]]; then
-  exec java ${JAVA_OPTS} -cp .:$(ls BOOT-INF/lib/*.jar|tr '\n' ':')BOOT-INF/classes zipkin.server.ZipkinServer
+  exec java ${JAVA_OPTS} -cp .:$(ls ${PWD}/BOOT-INF/lib/*.jar|tr '\n' ':')${PWD}/BOOT-INF/classes zipkin.server.ZipkinServer
 else
   exec java ${MODULE_OPTS} ${JAVA_OPTS} -cp . org.springframework.boot.loader.PropertiesLauncher
 fi

